### DISCUSSION TITLE
Catch a newly rendered x-for element on creation?

### DISCUSSION THREAD

**Discussion by noam-elbaz** [alpinejs/alpine#2211] [1. Help]
I have a x-for template that creates a list from an array.  
When I push a new item to the array, the x-for template renders the new element.

I would like to "catch" the newly added element at the moment of creation and apply some transitions to it like opacity transition (preferably with Tailwind animation classes, but regular CSS animations are ok too) 

Any suggestions how I can watch / observe a new x-for element when its being rendered for the first time?
Or how can I apply some basic animations / transitions?

(x-transition only works with x-show and it seems a bit ridiculous to add an x-show to then template)

Thanks.
N.

**Comment by SimoTod** [comment]
You probably don't need Alpine for the animation https://codepen.io/SimoTod/pen/qBXBeWO
